src/pathways/common/services/lms-client.service.ts

```
@@ -485,60 +485,37 @@ export class LmsClientService {
       );
-      return results;
-    };
 
-    const concurrency = LmsClientService.ENROLL_CONCURRENCY;
-    for (let i = 0; i < courseIds.length; i += concurrency) {
-      const chunk = courseIds.slice(i, i + concurrency);
-      const chunkResults = await runChunk(chunk);
-      alreadyEnrolledCount += chunkResults.filter((r) => r.alreadyEnrolled).length;
-    }
+      if (res.status >= 200 && res.status < 300) {
+        const alreadyEnrolledCount = res.data?.alreadyEnrolledCourseIds?.length || 0;
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)